### DISCUSSION TITLE
Crytocurrencies swap

### DISCUSSION THREAD

**Discussion by mingus52** [ghostfolio/ghostfolio#1699] [Ideas]
Hi,

My request is pretty simple, can you had the possibility to add a "Swap" capability for Cryptocurrencies ?

Example:  I buy BTC and "SWAP" those BTC for ETH.

If this is already possible, i didn't find the way in the software.

Best Regards

**Comment by dtslvr** [comment]
Hi @mingus52

This is not directly supported in Ghostfolio at the moment. However, you can still achieve the swap with one additional activity via a fiat currency:

* `SELL` `BTC` (via `USD`)
* `BUY` `ETH` (via `USD`)
